component/src/main/java/org/wso2/extension/siddhi/store/influxdb/InfluxDBStore.java

```diff
@@ -0,0 +1,565 @@
+
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.ANNOTATION_ELEMENT_PASSWORD;
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.ANNOTATION_ELEMENT_TABLE_NAME;
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.ANNOTATION_ELEMENT_URL;
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.ANNOTATION_ELEMENT_USERNAME;
+
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.DELETE_QUERY;
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.INFLUXQL_WHERE;
+import static org.wso2.extension.siddhi.store.influxdb.InfluxDBTableConstants.SELECT_QUERY;
+import static org.wso2.siddhi.core.util.SiddhiConstants.ANNOTATION_INDEX;
+import static org.wso2.siddhi.core.util.SiddhiConstants.ANNOTATION_STORE;
+
+/**
+ * Class representing the InfluxDB store implementation
+ */
+
+@Extension(
+        name = "influxdb",
+        namespace = "store",
+        description = "This extension assigns connection instructions to configure influxDB store. " +
+                "It also implements read-write operations on connected influxDB database.",
+        parameters = {
+                @Parameter(
+                        name = "url",
+                        description = " Url via which the InfluxDB data store is accessed",
+                        defaultValue = "http://localhost:8086 ",
+                        type = {DataType.STRING}
+                ),
+                @Parameter(
+                        name = "username",
+                        description = " The username used to access InfluxDB data store ",
+                        type = {DataType.STRING}
+                ),
+                @Parameter(
+                        name = "password",
+                        description = " The password used to access InfluxDB data store ",
+                        type = {DataType.STRING}
+                ),
+                @Parameter(
+                        name = "database",
+                        description = " The database to which the data should be entered. ",
+                        type = {DataType.STRING}
+                ),
+                @Parameter(name = "table.name",
+                        description = "The name with which the event table should be persisted in the store. If no " +
+                                "name is specified via this parameter, the event table is persisted with the same " +
+                                "name as the Siddhi table.",
+                        type = {DataType.STRING},
+                        optional = true,
+                        defaultValue = "The table name defined in the Siddhi App query."),
+        },
+        examples = {
+                @Example(
+                        syntax = " @Store(type = \"influxdb\",\n" +
+                                "   url = \"http://localhost:8086\",\n" +
```

Comment:
Is there any way to configure the connection params if any?

Reply:
https://github.com/wso2-extensions/siddhi-store-influxdb/pull/1/commits/902f3e82124dd61e28a18c062de43c6818e5793d